Sources/Enums/AnimationType.swift

```diff
@@ -121,8 +121,12 @@ extension AnimationType: IBEnum {
     case "moveby":
       self = .moveBy(x: params[safe: 0]?.toDouble() ?? 0, y: params[safe: 1]?.toDouble() ?? 0)
     case "scale":
-      self = .scale(fromX: params[safe: 0]?.toDouble() ?? 0, fromY: params[safe: 1]?.toDouble() ?? 0,
-                    toX: params[safe: 2]?.toDouble() ?? 1, toY: params[safe: 3]?.toDouble() ?? 1)
+      // We need to split this up or it results in `expression too complex to compile in a reasonable amount of time`
```

Comment:
I think you can drop the comment here

Reply:
Yeah, I just included it for the review 👍